.changeset/ten-peaches-tickle.md

```diff
@@ -0,0 +1,12 @@
+---
+"@builder.io/sdk-react-native": patch
```

Comment:
we can remove react-native from here as this feature doesn't apply

Reply:
Agreed